src/main/java/io/netty/incubator/codec/http3/Http3RequestStreamInboundHandler.java

```diff
@@ -109,6 +125,26 @@ protected void channelRead(@SuppressWarnings("unused") ChannelHandlerContext ctx
         frame.release();
     }
 
+    /**
+     * Called once a {@link QuicException} should be handled. By default, these exceptions are just logged
```

Comment:
nit: Committing to implementation details in the API are not generally recommended as we may change it later but some folks may depend on the behavior.

```suggestion
     * Handles a {@link QuicException} raised via {@link ChannelPipeline#exceptionCaught(ChannelHandlerContext,  Throwable)}.
```